### PR TITLE
docs: Refactors remaining enterprise topics, corrects relrefs

### PR DESCRIPTION
**Pull request by chri2547**
This PR refactors remaining enterprise topics and corrects relrefs.

### CODE REVIEW

File: docs/sources/administration/data-source-management/index.md [hunk 1]
```diff
@@ -98,3 +99,86 @@ Note that _all_ existing permissions created for the data source will be deleted
...
 
 <div class="clearfix"></div>
+
+## Query caching
+
+When query caching is enabled, Grafana temporarily stores the results of data source queries. When you or another user submit the exact same query again, the results will come back from the cache instead of from the data source (like Splunk or ServiceNow) itself.
+
+Query caching works for all backend data sources. You can enable the cache globally and configure the cache duration (also called Time to Live, or TTL).
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/administration/data-source-management/index.md:109]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

File: docs/sources/setup-grafana/configure-security/audit-grafana.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Auditing allows you to track important changes to your Grafana instance. By defa
 
 Only API requests or UI actions that trigger an API request generate an audit log.
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) version 7.3 and later, and [Grafana Cloud Advanced]({{< ref "/docs/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 7.3 and later, and [Grafana Cloud Advanced]({{< ref "/docs/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/setup-grafana/configure-security/audit-grafana.md:21]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 7.3 and later, and [Grafana Cloud Advanced](/docs/grafana-cloud).
```

File: docs/sources/setup-grafana/configure-security/configure-request-security.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ Request security allows you to limit requests from the Grafana server by targeti
 
 This can be used to limit access to internal systems that the server Grafana runs on can access but that users of Grafana should not be able to access. This feature does not affect traffic from the Grafana users browser.
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) version 7.4 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/docs/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 7.4 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/docs/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/setup-grafana/configure-security/configure-request-security.md:18]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 7.4 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

File: docs/sources/setup-grafana/configure-security/configure-team-sync.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ weight: 1000
 
 Team sync lets you set up synchronization between your auth providers teams and teams in Grafana. This enables LDAP, OAuth, or SAML users who are members of certain teams or groups to automatically be added or removed as members of certain teams in Grafana.
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) and [Grafana Cloud Advanced]({{< ref "/docs/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced]({{< ref "/docs/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/setup-grafana/configure-security/configure-team-sync.md:17]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud/).
```

File: docs/sources/administration/data-source-management/index.md [hunk 5]
```diff
@@ -46,7 +47,7 @@ To add a data source:
 
 Data source permissions allow you to restrict access for users to query a data source. For each data source there is a permission page that allows you to enable permissions and restrict query permissions to specific **Users** and **Teams**.
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/administration/data-source-management/index.md:50]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud).
```

File: docs/sources/administration/recorded-queries/index.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
 aliases:
   - /docs/grafana/latest/enterprise/recorded-queries/
+  - docs/grafana/latest/administration/recorded-queries/
```
Comment (gguillotte-grafana) [inline docs/sources/administration/recorded-queries/index.md:4]:
```suggestion
  - /docs/grafana/latest/administration/recorded-queries/
```

File: docs/sources/dashboards/create-reports/index.md [hunk 7]
```diff
@@ -76,15 +76,15 @@ Only organization administrators can create reports by default. You can customiz
 
 ### Choose template variables
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) version 7.5 and later behind the `reportVariables` feature flag, Grafana Enterprise version 8.0 and later without a feature flag, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 7.5 and later behind the `reportVariables` feature flag, Grafana Enterprise version 8.0 and later without a feature flag, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/dashboards/create-reports/index.md:79]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 7.5 and later behind the `reportVariables` feature flag, Grafana Enterprise version 8.0 and later without a feature flag, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

File: docs/sources/dashboards/create-reports/index.md [hunk 8]
```diff
@@ -76,15 +76,15 @@ Only organization administrators can create reports by default. You can customiz
...
 
 > **Note:** The query variables saved with a report might become of date if the results of that query change. For example, if your template variable queries for a list of hostnames and a new hostname is added, then it will not be included in the report. If that occurs, the selected variables must be manually updated in the report. If you select the `All` value for the template variable or if you keep the dashboard's original variable selection, then the report stays up-to-date as new values are added.
 
 ### Render a report with panels or rows set to repeat by a variable
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) version 8.0 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 8.0 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/dashboards/create-reports/index.md:87]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 8.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

File: docs/sources/dashboards/create-reports/index.md [hunk 9]
```diff
@@ -99,7 +99,7 @@ You can include dynamic dashboards with panels or rows, set to repeat by a varia
 
 ### Report time range
 
-> **Note:** You can set custom report time ranges in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) 7.2+ and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
+> **Note:** You can set custom report time ranges in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) 7.2+ and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/dashboards/create-reports/index.md:102]:
```suggestion
> **Note:** You can set custom report time ranges in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) 7.2+ and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

File: docs/sources/dashboards/create-reports/index.md [hunk 10]
```diff
@@ -123,7 +123,7 @@ If the time zone is set differently between your Grafana server and its remote i
 
 ### CSV export
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) 8+ with the [Grafana image renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) v3.0+, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) 8+ with the [Grafana image renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer) v3.0+, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/dashboards/create-reports/index.md:126]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) 8+ with the [Grafana image renderer plugin](/grafana/plugins/grafana-image-renderer) v3.0+, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

File: docs/sources/dashboards/create-reports/index.md [hunk 11]
```diff
@@ -135,7 +135,7 @@ A background job runs every 10 minutes and removes temporary CSV files. You can
 
 ### Scheduling
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) version 8.0 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 8.0 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/dashboards/create-reports/index.md:138]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 8.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

File: docs/sources/dashboards/create-reports/index.md [hunk 12]
```diff
@@ -158,7 +158,7 @@ When you schedule a report with a monthly frequency, and set the start date betw
 
 #### Send a test email
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) version 7.0 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 7.0 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/dashboards/create-reports/index.md:161]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 7.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

File: docs/sources/dashboards/create-reports/index.md [hunk 13]
```diff
@@ -171,19 +171,19 @@ The last saved version of the report will be sent to selected emails. You can us
 
 ### Pause a report
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) version 8.0 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 8.0 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/dashboards/create-reports/index.md:174]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 8.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

File: docs/sources/dashboards/create-reports/index.md [hunk 14]
```diff
@@ -171,19 +171,19 @@ The last saved version of the report will be sent to selected emails. You can us
...
 
 You can pause sending reports from the report list view by clicking the pause icon. The report will not be sent according to its schedule until it is resumed by clicking the resume button on the report row.
 
 ### Add multiple dashboards to a report
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) version 9.0 and later, and [Grafana Cloud Pro and Advanced]({{< relref "/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 9.0 and later, and [Grafana Cloud Pro and Advanced]({{< relref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/dashboards/create-reports/index.md:180]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 9.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

File: docs/sources/dashboards/create-reports/index.md [hunk 15]
```diff
@@ -171,19 +171,19 @@ The last saved version of the report will be sent to selected emails. You can us
...
 
 You can add more than one dashboard to a report. Additional dashboards will be rendered as new pages in the same PDF file, or additional images if you chose to embed images in your report email. You cannot add the same dashboard to a report multiple times.
 
 ### Embed a dashboard as an image into a report
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) version 9.0 and later, and [Grafana Cloud Pro and Advanced]({{< relref "/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 9.0 and later, and [Grafana Cloud Pro and Advanced]({{< relref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/dashboards/create-reports/index.md:186]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 9.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

File: docs/sources/dashboards/create-reports/index.md [hunk 16]
```diff
@@ -238,7 +238,7 @@ font_italic = DejaVuSansCondensed-Oblique.ttf
 
 ## Report settings
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../enterprise/" >}}) version 7.2 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 7.2 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/grafana-cloud" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/dashboards/create-reports/index.md:241]:
```suggestion
> **Note:** Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) version 7.2 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
```

**Review by gguillotte-grafana — CHANGES_REQUESTED** [review]
Callouts to the Cloud docs should use Markdown links instead of Hugo refs/relrefs.

**Comment by gguillotte-grafana** [comment]
Line 20 of docs/sources/introduction/grafana-enterprise.md has a Hugo ref to `/docs/grafana-cloud/reference/enterprise-features` - the Hugo ref shortcode to the Cloud docs won't work at all, but there's also no content at https://grafana.com/docs/grafana-cloud/reference/enterprise-features and I don't know where that should point.

**Comment by chri2547** [comment]
> Line 20 of docs/sources/introduction/grafana-enterprise.md has a Hugo ref to `/docs/grafana-cloud/reference/enterprise-features` - the Hugo ref shortcode to the Cloud docs won't work at all, but there's also no content at https://grafana.com/docs/grafana-cloud/reference/enterprise-features and I don't know where that should point.

Removed that link.

**Review by gguillotte-grafana — COMMENTED** [review]
LGTM - should docs/sources/introduction/grafana-enterprise.md be docs/sources/introduction/grafana-enterprise/index.md? Not a blocking concern, but similar to what came up in the data sources refactor.

**Review by gguillotte-grafana — APPROVED** [review]

**Comment by chri2547** [comment]
> LGTM - should docs/sources/introduction/grafana-enterprise.md be docs/sources/introduction/grafana-enterprise/index.md? Not a blocking concern, but similar to what came up in the data sources refactor.

Yes, I still need to do that and will handle it outside of this PR. Slowly making my way through the repo.

**Review by torkelo — APPROVED** [review]

**Pull request merged**